client/src/App.js

```diff
@@ -40,34 +34,34 @@ function App() {
           />
         </Navbar.Brand>
 
-        <Nav className='mr.auto'>
-          <Link to='/dash'>Dash</Link>
-          <Link to='/categories'>Categories</Link>
-        </Nav>
-        <Navbar.Collapse className="justify-content-end">
+        <Navbar.Collapse className='justify-content-end'>
           <Nav className='mr.auto'>
             {document.cookie ? (
-              <Button onClick={logout}>Logout</Button>
+              <>
+                <Nav className='mr.auto'>
+                  <Link to='/dash'>Dash</Link>
+                  <Link to='/categories'>Categories</Link>
+                  <Button onClick={logout} href='/login'>Logout</Button>
+                </Nav>
+              </>
             ) : (
-                <>
-                  <Link to='/login'>Login</Link>
-                  <Link to='/register'>Register</Link>
-                </>
-              )}
+              <>
+                <Link to='/login'>Login</Link>
+                <Link to='/register'>Register</Link>
+              </>
+            )}
           </Nav>
         </Navbar.Collapse>
       </Navbar>
 
       <Switch>
-        <Route exact path='/'>
-          <Home />
-        </Route>
```

Comment:
Is there a particular reason why you're getting rid of this? If you're going to build the landing page, it might be useful to have it there for testing :)